packages/sources/index.js

```diff
@@ -0,0 +1,68 @@
+const SOURCES = {
+  CCN: "conventions_collectives",
+  CDT: "code_du_travail",
+  SHEET_SP: "fiches_service_public",
+  SHEET_MT: "fiches_ministere_travail",
+  THEMES: "themes",
+  TOOLS: "outils",
+  LETTERS: "modeles_de_courriers",
+  FAQ: "faq",
+  SNIPPET: "snippet",
+  IDCC: "idcc",
+  SHEET: "fiches",
+  EXTERNALS: "external"
+};
+
+// mapping elastic search source type -> route name
+const routeBySource = {
+  [SOURCES.FAQ]: "question",
+  [SOURCES.SHEET_SP]: "fiche-service-public",
+  [SOURCES.SHEET_MT]: "fiche-ministere-travail",
+  [SOURCES.CDT]: "code-du-travail",
+  [SOURCES.LETTERS]: "modeles-de-courriers",
+  [SOURCES.THEMES]: "themes",
+  [SOURCES.TOOLS]: "outils",
+  [SOURCES.IDCC]: "idcc",
+  [SOURCES.CCN]: "convention-collective",
+  [SOURCES.EXTERNALS]: "external"
```

Comment:
not sure if it should be a in "routeBySource"

Reply:
routeBySource -> routes ?
labelBySource -> labels ?